### PR TITLE
feat: add creator and token count to tokens API

### PR DESCRIPTION
**Pull request by tv3636**

### CODE REVIEW

File: packages/indexer/src/api/endpoints/tokens/get-tokens/v6.ts [hunk 1]
```diff
@@ -1125,11 +1129,13 @@ export const getTokensV6Options: RouteOptions = {
               : undefined,
             media: r.media,
             kind: r.kind,
+            creator: r.creator ? fromBuffer(r.creator) : null,
```
Comment (nofir) [inline packages/indexer/src/api/endpoints/tokens/get-tokens/v6.ts:1132]: this should be on the collection obj

File: packages/indexer/src/api/endpoints/tokens/get-tokens/v6.ts [hunk 2]
```diff
@@ -1125,11 +1129,13 @@ export const getTokensV6Options: RouteOptions = {
...
             isFlagged: Boolean(Number(r.is_flagged)),
             lastFlagUpdate: r.last_flag_update ? new Date(r.last_flag_update).toISOString() : null,
             lastFlagChange: r.last_flag_change ? new Date(r.last_flag_change).toISOString() : null,
             supply: !_.isNull(r.supply) ? r.supply : null,
             remainingSupply: !_.isNull(r.remaining_supply) ? r.remaining_supply : null,
+            tokenCount: r.token_count,
```
Comment (nofir) [inline packages/indexer/src/api/endpoints/tokens/get-tokens/v6.ts:1138]: this should be on the collection obj

**Review by nofir — CHANGES_REQUESTED** [review]

File: packages/indexer/src/api/endpoints/tokens/get-tokens/v6.ts [hunk 3]
```diff
@@ -242,6 +242,8 @@ export const getTokensV6Options: RouteOptions = {
               name: Joi.string().allow("", null),
               image: Joi.string().allow("", null),
               slug: Joi.string().allow("", null),
+              creator: Joi.string().lowercase().pattern(regex.address).required(),
```
Comment (nofir) [inline packages/indexer/src/api/endpoints/tokens/get-tokens/v6.ts:245]: why is there required()? also, should allow null

**Review by nofir — CHANGES_REQUESTED** [review]

File: packages/indexer/src/api/endpoints/tokens/get-tokens/v6.ts (hunk 3 above)
Comment (tv3636) [reply packages/indexer/src/api/endpoints/tokens/get-tokens/v6.ts]: yeah not sure, fixed

**Review by nofir — APPROVED** [review]

**Pull request merged**